### PR TITLE
Adding privacy 2020 chapter

### PR DESCRIPTION
**Pull request by ydimova**
Adding privacy chapter text. Progress on #913

### CODE REVIEW

File: src/content/en/2020/privacy.md [hunk 1]
```diff
@@ -12,12 +12,166 @@ translators: []
 discuss: 2046
```
Comment (rviscomi) [inline src/content/en/2020/privacy.md:12]: @ydimova can you also add a bio for yourself to be included at the bottom of the chapter? (see TODO above)

**Review by rviscomi — COMMENTED** [review]
Thanks Yana, I'm excited to see this go live!

**Comment by rviscomi** [comment]
Oh I just noticed this PR is merging in to `privacy_2020`. Was that intentional? If this is the final draft could you edit the PR to merge into `main` instead?

**Comment by bazzadp** [comment]
FYI, I've reviewed the charts and tweak a few of them (changed to default size of 600 by 371, checked colours, checked titles are in sentence case...etc.).

I've then staged the chapter here so you can see what it looks like: https://20201207t214858-dot-webalmanac.uk.r.appspot.com/en/2020/privacy. Please review.

The build and test job is failing as your fallback images do not exist. Let me know if you need help generating them.

File: src/content/en/2020/privacy.md [hunk 2]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+featured_stat_label_2: Largest domains that set cookies on the biggest percentage of websites are Google-owned.
+featured_stat_3: 44.8%
+featured_stat_label_3: Sites have a privacy policy
 unedited: true
 ---
+
+
+## Introduction
+
+This chapter of the Web Almanac gives an overview of the current state of privacy on the web. This topic has been increasing in popularity recently and has raised awareness on the users’ side. The need for guidelines has been met with various regulations (such as [GDPR](https://gdpr-info.eu/) in Europe,  [LGPD](https://lgpd-brazil.info/) in Brazil, [CCPA](https://leginfo.legislature.ca.gov/faces/codes_displayText.xhtml?division=3.&part=4.&lawCode=CIV&title=1.81.5) in California…), that aim to increase the accountability of data processors and their transparency  towards users. In this chapter, we discuss the prevalence of online tracking with different techniques and the adoption rate of cookie consent banners and privacy policies by websites.
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:28]:
```suggestion
This chapter of the Web Almanac gives an overview of the current state of privacy on the web. This topic has been increasing in popularity recently and has raised awareness on the users’ side. The need for guidelines has been met with various regulations (such as [GDPR](https://gdpr-info.eu/) in Europe,  [LGPD](https://lgpd-brazil.info/) in Brazil, [CCPA](https://leginfo.legislature.ca.gov/faces/codes_displayText.xhtml?division=3.&part=4.&lawCode=CIV&title=1.81.5) in California…). These aim to increase the accountability of data processors and their transparency  towards users. In this chapter, we discuss the prevalence of online tracking with different techniques and the adoption rate of cookie consent banners and privacy policies by websites.
```

File: src/content/en/2020/privacy.md [hunk 3]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+
+
+## Online Tracking
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:31]: Nit - you sometimes have a newline after heading, sometimes don't.

```suggestion
## Online Tracking

```

File: src/content/en/2020/privacy.md [hunk 4]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+Third-party trackers collect user data to build up profiles of the user’s behavior to be monetized for advertising purposes. This raises privacy concerns with users on the web, which resulted in the emergence of various tracking protections. However, as we will see in this section, online tracking is still widely used. Not only does it have a negative impact on privacy, online tracking has a [huge impact on the environment](https://gerrymcgovern.com/calculating-the-pollution-cost-of-website-analytics-part-1/) and avoiding it can lead to [better performance](https://twitter.com/fr3ino/status/1000166112615714816). 
+We examine the prominence of the most common types of third-party tracking, namely by means of third-party cookies and the use of fingerprinting. Online tracking is not limited to just these two techniques, new ones keep arising to circumvent existing countermeasures. 
+
+
+### Third-party trackers
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:36]:
```suggestion
### Third-party trackers

```

File: src/content/en/2020/privacy.md [hunk 5]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+We use [WhoTracksMe](https://whotracks.me/)’s tracker list to determine the percentage of websites that issue a request to a tracker. As in the following figure, we find that on roughly 93% of websites, at least one tracker is present. 
+
+{{ figure_markup(
+  image="websites_that_load_trackers.png",
+  caption="Websites including at least one tracker",
+  description="This figure shows the percentage of websites that load at least one tracker.",
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:42]: Description should describe the chart in detail:

```suggestion
  description="Bar chart showing that 92.94% of desktop websites and 92.97% of mobile websites load trackers.",
```

File: src/content/en/2020/privacy.md [hunk 6]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=1325818112&format=interactive",
+  sheets_gid="1591448294"
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:44]: Do we have the sql for this stat? Can't see it in https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020/10_Privacy/

File: src/content/en/2020/privacy.md [hunk 7]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  )
+}}
+
+We examine the largest (most widely used) trackers and plot the prevalence of the 10 most popular ones.
+
+{{ figure_markup(
+  image="biggest_third-party_trackers.png",
+  caption="Top 10 Trackers",
+  description="This figure shows the prevalence of the 10 most popular trackers used on mobile and desktop clients.",
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:53]:
```suggestion
  description="Bar chart showing the prevalence of the 10 most popular trackers used on mobile and desktop clients. There is little difference between desktop and mobile and mobile has 65.9% for google_analytics, 65.5% for googleapis.com, 63.3% for gstatic, 58.3% for google_fonts, 50.0% for doubleclick, 47.6% for google, 42.4% for google_tag_manage, 30.9% for facebook, 19.2% for google_adservices, and 13.1% for cloudflare.",
```

File: src/content/en/2020/privacy.md [hunk 8]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=850649042&format=interactive",
+  sheets_gid="1677398038"
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:55]:
```suggestion
  sheets_gid="1677398038",
  sql_file="top100_cookies_set_from_header.sql"
```

File: src/content/en/2020/privacy.md [hunk 9]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  )
+}}
+
+The largest player on the online tracking market is definitely Google, with eight of  its tracking domains present in the top 10 trackers and prevalent on at least 70% of websites. Following are Facebook and Cloudflare.
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:59]:
```suggestion
The largest player on the online tracking market is definitely Google, with eight of its tracking domains present in the top 10 trackers and prevalent on at least 70% of websites. Following are Facebook and Cloudflare. 
```

File: src/content/en/2020/privacy.md [hunk 10]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+}}
+
+The largest player on the online tracking market is definitely Google, with eight of  its tracking domains present in the top 10 trackers and prevalent on at least 70% of websites. Following are Facebook and Cloudflare. 
+
+WhoTracksMe’s tracker list also defines categories that the trackers belong to. The following figure shows the distribution of the different categories for the 100 largest trackers.
+
+{{ figure_markup(
+  image="tracker_categories.png",
+  caption="Categories of the 100 most popular trackers",
+  description="This figure shows distribution of the top 100 trackers on the web.",
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:66]:
```suggestion
  description="Bar chart showing distribution of the top 100 trackers on the web with 56 for advertising, 11 for cdn, 9 for site_analytics, 6 for both social media and misc, 3 for both essential and customer_help, 2 for both audio and video and 1 for both comments and undefined.",
```

File: src/content/en/2020/privacy.md [hunk 11]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=1117413918&format=interactive",
+  sheets_gid="1431872451",
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:68]: Again I can't see the SQL for this data.

File: src/content/en/2020/privacy.md [hunk 12]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  )
+}}
+
+Nearly 60% of the most popular trackers are advertising-related. This could be due to the profitability of the online advertising market.
+
+### Cookies
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:74]:
```suggestion
### Cookies

```

File: src/content/en/2020/privacy.md [hunk 13]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+|mobile|	youtube.com|	VISITOR_INFO1_LIVE|10%
+|mobile	|youtube.com|	YSC|10%
+|mobile|	google.com|	NID|10%
+|mobile|	youtube.com	|GPS|	8%
+|mobile|	doubleclick.net	|DSID|	7%
+|mobile|	yandex.ru|	yandexuid|	6%
+|mobile|	yandex.ru|	i|	6%
+
+<figcaption>{{ figure_link(caption="Top cookies", sheets_gid="732942035", sql_file="top100_cookies_set_from_header.sql") }}</figcaption>
+</figure>
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:102]: Suggest splitting this into two tables, and also using HTML so can apply numeric class and align the percentages.

```suggestion
<figure>
  <table>
    <thead>
      <tr>
        <th>Domain</th>
        <th>Cookie Name</th>
        <th>Websites present on</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>doubleclick.net</td>
        <td>test_cookie</td>
        <td class="numeric">24%</td>
      </tr>
      <tr>
        <td>facebook.com</td>
        <td>fr</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>VISITOR_INFO1_LIVE</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>YSC</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>IDE</td>
        <td class="numeric">9%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>unknown</td>
        <td class="numeric">9%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>GPS</td>
        <td class="numeric">9%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>unknown</td>
        <td class="numeric">8%</td>
      </tr>
      <tr>
        <td>google.com</td>
        <td>NID</td>
        <td class="numeric">6%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>unknown</td>
        <td class="numeric">6%</td>
      </tr>
    </tbody>
  </table>
  <figcaption>{{ figure_link(caption="Top cookies on desktop sites", sheets_gid="732942035", sql_file="top100_cookies_set_from_header.sql") }}</figcaption>
</figure>

<figure>
  <table>
    <thead>
      <tr>
        <th>Domain</th>
        <th>Cookie Name</th>
        <th>Websites present on</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>doubleclick.net</td>
        <td>test_cookie</td>
        <td class="numeric">32%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>IDE</td>
        <td class="numeric">21%</td>
      </tr>
      <tr>
        <td>facebook.com</td>
        <td>fr</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>VISITOR_INFO1_LIVE</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>YSC</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>google.com</td>
        <td>NID</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>GPS</td>
        <td class="numeric">8%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>DSID</td>
        <td class="numeric">7%</td>
      </tr>
      <tr>
        <td>yandex.ru</td>
        <td>yandexuid</td>
        <td class="numeric">6%</td>
      </tr>
      <tr>
        <td>yandex.ru</td>
        <td>i</td>
        <td class="numeric">6%</td>
      </tr>
    </tbody>
  </table>
  <figcaption>{{ figure_link(caption="Top cookies on mobile sites", sheets_gid="732942035", sql_file="top100_cookies_set_from_header.sql") }}</figcaption>
</figure>
```

File: src/content/en/2020/privacy.md [hunk 14]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+
+As found previously, Google’s  tracking domain ‘doubleclick.net’  sets cookies on roughly one fourth of websites on a mobile client and one third of all  websites on a desktop client. Again, nine out of the ten most popular cookies on desktop client and 7 on mobile are set by a Google domain. This is a lower bound for the amount of websites the cookie is set on, since we are only conting cookies set via an HTTP header, a large amount of tracking cookies is set by using third-party scripts.
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:104]:
```suggestion
As found previously, Google’s tracking domain ‘doubleclick.net’ sets cookies on roughly a quarter of websites on a mobile client and one third of websites on a desktop client. Again, nine out of the ten of the most popular cookies on desktop client and 7 on mobile are set by a Google domain. This is a lower bound for the amount of websites the cookie is set on, since we are only counting cookies set via an HTTP header, a large amount of tracking cookies are set by using third-party scripts.
```

File: src/content/en/2020/privacy.md [hunk 15]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+
+### Fingerprinting
+
+Another widely-used tracking technique is fingerprinting, which consists of collecting different kinds of information about the user with the goal of building a unique “fingerprint” for them. Different types of fingerprinting are used on the web by trackers. Browser fingerprinting uses characteristics specific to the browser of the user, relying on the fact that the chance of another user having the exact same browser information is fairly small. In our crawl, we examined the presence of  the [FingerprintJS](https://fingerprintjs.com/) library, which provides browser fingerprinting as a service.  
+
+{{ figure_markup(
+  image="websites_with_fingerprint.js_library.png",
+  caption="Websites using FingerprintJS",
+  description="This figure shows the percentage of websites using FingerprintJS.",
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:113]:
```suggestion
  description="Barchart showing 0.17% of desktop sites and 0.18% of mobile sites use FingerprintJS.",
```

File: src/content/en/2020/privacy.md [hunk 16]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=1556252953&format=interactive",
+  sheets_gid="222110824",
+    sql_file="percent_of_websites_with_fingerprinting.sql "
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:116]:
```suggestion
  sql_file="percent_of_websites_with_fingerprinting.sql "
```

File: src/content/en/2020/privacy.md [hunk 17]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  )
+}}
+Although the library is present on less than 1% of websites, the main issue with fingerprinting is its persistent nature.  Furthermore, FingerprintJS is not the only attempt at fingerprinting.  Other libraries, tools and native code can also serve this purpose.
+
+## Consent Management Platforms
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:121]:
```suggestion
## Consent Management Platforms

```

File: src/content/en/2020/privacy.md [hunk 18]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+{{ figure_markup(
+  image="websites_with_fingerprint.js_library.png",
+  caption="Websites using FingerprintJS",
+  description="This figure shows the percentage of websites using FingerprintJS.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=1556252953&format=interactive",
+  sheets_gid="222110824",
+    sql_file="percent_of_websites_with_fingerprinting.sql "
+  )
+}}
+Although the library is present on less than 1% of websites, the main issue with fingerprinting is its persistent nature.  Furthermore, FingerprintJS is not the only attempt at fingerprinting.  Other libraries, tools and native code can also serve this purpose.
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:119]:
```suggestion

Although the library is present on less than 1% of websites, the main issue with fingerprinting is its persistent nature.  Furthermore, FingerprintJS is not the only attempt at fingerprinting.  Other libraries, tools and native code can also serve this purpose.
```

Comment (bazzadp) [inline src/content/en/2020/privacy.md:119]: Way less than 1% so suggest rewording:

```suggestion
Although the library is present on a small percentage of websites, the main issue with fingerprinting is its persistent nature.  Furthermore, FingerprintJS is not the only attempt at fingerprinting.  Other libraries, tools and native code can also serve this purpose.
```

File: src/content/en/2020/privacy.md [hunk 19]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+
+## Consent Management Platforms
+Cookie consent banners are being used by websites for years now, to allow users to specify their third-party cookie choices as well as increase transparency towards them. While a lot of websites opt for using their own implementation of cookie banners, third-party solutions called ‘Consent Management Platforms’ have recently emerged. The platforms provide an easy way for websites to collect user’s consent for different types of cookies. We see that 4.4% of websites use some consent management platform  to provide cookie choices on desktop clients, and 4% on mobile clients.
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:122]: Not all banners allow cooking choice. Some are still (dubiously!) going with "By using this website you consent..."

```suggestion
Cookie consent banners have become common now increase transparency towards cookies and often allowing users to specify their cookie choices. While a lot of websites opt for using their own implementation of cookie banners, third-party solutions called ‘Consent Management Platforms’ have recently emerged. The platforms provide an easy way for websites to collect user’s consent for different types of cookies. We see that 4.4% of websites use some consent management platform to provide cookie choices on desktop clients, and 4.0% on mobile clients. 
```

File: src/content/en/2020/privacy.md [hunk 20]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+}}
+Although the library is present on less than 1% of websites, the main issue with fingerprinting is its persistent nature.  Furthermore, FingerprintJS is not the only attempt at fingerprinting.  Other libraries, tools and native code can also serve this purpose.
+
+## Consent Management Platforms
+Cookie consent banners are being used by websites for years now, to allow users to specify their third-party cookie choices as well as increase transparency towards them. While a lot of websites opt for using their own implementation of cookie banners, third-party solutions called ‘Consent Management Platforms’ have recently emerged. The platforms provide an easy way for websites to collect user’s consent for different types of cookies. We see that 4.4% of websites use some consent management platform  to provide cookie choices on desktop clients, and 4% on mobile clients. 
+
+{{ figure_markup(
+  image="websites_with_consent_management _platform.png",
+  caption="Websites using a consent management platform",
+  description="This figure shows the ahre of websites that are using a popular consent management platform.",
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:127]:
```suggestion
  description="Bar chart showing 4.4% of desktop sites and 4.0% of mobile sites use a Consent Management Platform.",
```

File: src/content/en/2020/privacy.md [hunk 21]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=2025542332&format=interactive",
+  sheets_gid="1910033502",
+    sql_file="percent_of_websites_with_cmp.sql"
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:130]:
```suggestion
  sql_file="percent_of_websites_with_cmp.sql"
```

File: src/content/en/2020/privacy.md [hunk 22]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  )
+}}
+{{ figure_markup(
+  image="consent_management_platform_popularity.png",
+  caption="Popularity of consent management platform",
+  description="This figure shows the popularity of each consent management platform.",
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:136]:
```suggestion
  description="Bar chart showing popular consent management platforms from Osano at 1.6%, Quantcast Choice at 1.0%, Cookiebot and OneTrust at 0.4%, Iubenda at 0.3%, Crownpeak, Didomi, and TrustArc all at 0.1%, CIVIC, Cookie Script, CookieHub, Termly, Uniconsent, CookieYes, eucookie.eu, Seers, and Metomic all at approximately 0.0%.",
```

File: src/content/en/2020/privacy.md [hunk 23]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=341496718&format=interactive",
+  sheets_gid="1104760876",
+    sql_file="percent_of_websites_using_each_cmp.sql"
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:139]:
```suggestion
  sql_file="percent_of_websites_using_each_cmp.sql"
```

File: src/content/en/2020/privacy.md [hunk 24]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  )
+}}
+
+When looking at the popularity of the different consent management solutions, we can see that Osano and Quantcast choice are the leading platforms.
+
+#### IAB Europe’s Transparency Consent Framework
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:145]:
```suggestion
#### IAB Europe’s Transparency Consent Framework

```

File: src/content/en/2020/privacy.md [hunk 25]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+IAB Europe proposed a [Transparency Consent Framework](https://iabeurope.eu/transparency-consent-framework/) (TCF) as a GDPR-compliant solution to obtain users' consent about their digital advertising preferences. The implementation provides an industry standard for communication between publishers and advertisers about consumer consent.
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:146]: Who are IAB Europe? Can you introduce them?

File: src/content/en/2020/privacy.md [hunk 26]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+{{ figure_markup(
+  image="adoption_of_the_TCF_banner.png",
+  caption="Adoption rate of TCF banner",
+  description="This figure shows the percentage of websites using IAB Europe's TCF banner.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=341275612&format=interactive",
+  sheets_gid="2077755325",
+    sql_file="percent_of_websites_with_iab_tcf_banner.sql"
+  )
+}}
+While our results show that the TCF banner is not yet the ‘industry standard’, it is a step in the right direction. Considering the main target group of IAB Europe is in fact European publishers, having an adoption rate on 1.5% of websites on desktop client and 1.4% on mobile is not too bad.
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:156]:
```suggestion
  sql_file="percent_of_websites_with_iab_tcf_banner.sql"
  )
}}

While our results show that the TCF banner is not yet the ‘industry standard’, it is a step in the right direction. Considering the main target group of IAB Europe is in fact European publishers, having an adoption rate on 1.5% of websites on desktop client and 1.4% on mobile is not too bad. 
```

File: src/content/en/2020/privacy.md [hunk 27]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+}}
+
+When looking at the popularity of the different consent management solutions, we can see that Osano and Quantcast choice are the leading platforms.
+
+#### IAB Europe’s Transparency Consent Framework
+IAB Europe proposed a [Transparency Consent Framework](https://iabeurope.eu/transparency-consent-framework/) (TCF) as a GDPR-compliant solution to obtain users' consent about their digital advertising preferences. The implementation provides an industry standard for communication between publishers and advertisers about consumer consent. 
+{{ figure_markup(
+  image="adoption_of_the_TCF_banner.png",
+  caption="Adoption rate of TCF banner",
+  description="This figure shows the percentage of websites using IAB Europe's TCF banner.",
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:150]:
```suggestion
  description="Bar chart showing that 1.5% of desktop sites and 1.4% of mobile sites have implemented IAB Europe's TCF banner.",
```

File: src/content/en/2020/privacy.md [hunk 28]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+}}
+While our results show that the TCF banner is not yet the ‘industry standard’, it is a step in the right direction. Considering the main target group of IAB Europe is in fact European publishers, having an adoption rate on 1.5% of websites on desktop client and 1.4% on mobile is not too bad. 
+
+## Privacy Policies
+Privacy policies are widely used by websites to increase transparency towards users about data collection practices. In our crawl, we searched for keywords indicating the presence of a privacy policy text on each visited website. 
+
+{{ figure_markup(
+  image="websites_with_privacy_link.png",
+  caption="Websites that have a privacy policy",
+  description="This figure shows the percentage of websites that indicate to have implemented a privacy policy text.",
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:164]:
```suggestion
  description="Bar chart showing that 44.8% of desktop sites and 42.3% of mobile sites have a privacy link.",
```

File: src/content/en/2020/privacy.md [hunk 29]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=329249623&format=interactive",
+  sheets_gid="495362514",
+    sql_file="percent_of_websites_with_privacy_links.sql "
+  )
+}}
+The results show that almost half of the websites in the dataset have included a privacy policy, which is positive. However, studies have shown that the majority of internet users do not bother reading privacy policies and when they do, they lack understanding due to the length and complexity of most privacy policy texts.
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:170]:
```suggestion
  sql_file="percent_of_websites_with_privacy_links.sql "
  )
}}

The results show that almost half of the websites in the dataset have included a privacy policy, which is positive. However, studies have shown that the majority of internet users do not bother reading privacy policies and when they do, they lack understanding due to the length and complexity of most privacy policy texts.
```

File: src/content/en/2020/privacy.md [hunk 30]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+
+## Conclusion
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:172]:
```suggestion
## Conclusion

```

File: src/content/en/2020/privacy.md [hunk 31]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+  caption="Adoption rate of TCF banner",
+  description="This figure shows the percentage of websites using IAB Europe's TCF banner.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=341275612&format=interactive",
+  sheets_gid="2077755325",
+    sql_file="percent_of_websites_with_iab_tcf_banner.sql"
+  )
+}}
+While our results show that the TCF banner is not yet the ‘industry standard’, it is a step in the right direction. Considering the main target group of IAB Europe is in fact European publishers, having an adoption rate on 1.5% of websites on desktop client and 1.4% on mobile is not too bad. 
+
+## Privacy Policies
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:158]:
```suggestion
## Privacy Policies

```

File: src/content/en/2020/privacy.md [hunk 32]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+Privacy policies are widely used by websites to increase transparency towards users about data collection practices. In our crawl, we searched for keywords indicating the presence of a privacy policy text on each visited website.
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:159]: The cynic in me says this (and the cookie banner above) are not being done for transparency reasons!

```suggestion
Privacy policies are widely used by websites to met legal obligations and increase transparency towards users about data collection practices. In our crawl, we searched for keywords indicating the presence of a privacy policy text on each visited website. 
```

File: src/content/en/2020/privacy.md [hunk 33]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+
+## Conclusion
+This chapter gives an overview of the current state of privacy on the web. Third-party tracking remains prominent on both desktop and mobile clients, with Google tracking the largest percentage of websites. Consent management platforms are used on a small percentage of websites, however a lot of websites implement their own cookie consent banners. Lastly, roughly half of the websites include a privacy policy, which benefits greatly transparency towards users about data processing practices.  This is undoubtedly a step forward but there is a lot to be done. Most of the time, privacy policies are hard to read and understand and cookie consent banners manipulate users into consent. For the web to truly respect users, privacy has to be a part of conception, not an afterthought. Regulations are a good thing and it is reassuring to see them appearing worldwide but, more than that and financial sanctions, Privacy by Design should be the norm, rather than deploying texts and tools in order to avoid fines.
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:173]:
```suggestion
This chapter gives an overview of the current state of privacy on the web. Third-party tracking remains prominent on both desktop and mobile clients, with Google tracking the largest percentage of websites. Consent Management Platforms are used on a small percentage of websites, however a lot of websites implement their own cookie consent banners. Lastly, roughly half of the websites include a privacy policy, which benefits greatly transparency towards users about data processing practices. This is undoubtedly a step forward but there is a lot to be done. Often, privacy policies are hard to read and understand and cookie consent banners manipulate users into consent. For the web to truly respect users, privacy has to be a part of conception, not an afterthought. Regulation is a good thing and it is reassuring to see an increase in privacy regulation worldwide but, more than that and financial sanctions, [privacy by design](https://en.wikipedia.org/wiki/Privacy_by_design) should be the norm, rather than deploying policies and tools in order to avoid fines.
```

File: src/content/en/2020/privacy.md [hunk 34]
```diff
@@ -12,12 +12,166 @@ translators: []
...
+
+
+
+
```
Comment (bazzadp) [inline src/content/en/2020/privacy.md:177]:
```suggestion
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
That was really interesting and I think this could be a key chapter for us. I'd love to get this merged, and hopefully even edited for launch.

I've given a number of pieced os feedback. Small hint for you - use the [Files tab](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1657/files) and the "Add suggestions to batch" buttons to accept each piece of feedback you are happy to take, and then submit in one go - it's a lot quicker than accepting them one by one.

**Comment by bazzadp** [comment]
I added the images and made the easy changes. Left the others for you to review. Think it's in a good stage to merge after that unless you're still writing any more?

File: src/content/en/2020/privacy.md (hunk 6 above)
Comment (ydimova) [reply src/content/en/2020/privacy.md:43]: Didn't add the queries for online tracking since I was using an external dataset (WhoTracksMe)

File: src/content/en/2020/privacy.md (hunk 18 above)
Comment (bazzadp) [reply src/content/en/2020/privacy.md]: @ydimova did you see this comment?

File: src/content/en/2020/privacy.md (hunk 19 above)
Comment (bazzadp) [reply src/content/en/2020/privacy.md]: @ydimova did you see this comment?

File: src/content/en/2020/privacy.md (hunk 32 above)
Comment (bazzadp) [reply src/content/en/2020/privacy.md]: @ydimova did you see this comment?

File: src/content/en/2020/privacy.md (hunk 18 above)
Comment (bazzadp) [reply src/content/en/2020/privacy.md]: Never mind - forgot to `git pull`. See you changed it.

File: src/content/en/2020/privacy.md (hunk 19 above)
Comment (bazzadp) [reply src/content/en/2020/privacy.md]: Ditto.

**Review by bazzadp — APPROVED** [review]
